yw_website/yw_website/apps/website/tests.py

```diff
@@ -319,3 +320,156 @@ def test_no_repeating_tags(self):
+            },
+            {
+                "dataId": 2,
+                "inProgramBlock": 1,
+                "name": "dataName2",
+                "qualifiedName": "dataQualifiedName2",
+            },
+        ]
+        self.data["port"] = [
+            {
+                "portId": 1,
+                "onProgramBlock": 1,
+                "data": 1,
+                "name": "PortOne",
+                "qualifiedName": "portOneQualifiedName",
+                "alias": None,
+                "uriTemplate": None,
+                "inPort": True,
+                "outPort": False,
+            },
+            {
+                "portId": 2,
+                "onProgramBlock": 2,
+                "data": 2,
+                "name": "PortTwo",
+                "qualifiedName": "portTwoQualifiedName",
+                "alias": "aliasTwo",
+                "uriTemplate": "uriTwo",
+                "inPort": False,
+                "outPort": True,
+            },
+        ]
+        self.data["channel"] = [
+            {
+                "channelId": 1,
+                "inPort": 1,
+                "outPort": 2,
+                "data": 1,
+                "isInflow": True,
+                "isOutflow": False,
+            }
+        ]
+        self.data["uriVariable"] = [
+            {"uriVariableId": 1, "port": 1, "name": "urivarname"}
+        ]
+        self.data["resource"] = [
+            {
+                "resourceId": 1,
+                "data": 1,
+                "uri": "file_uri_1",
+                "name": "file_name_1",
+                "checksum": str(uuid.uuid1()),
+                "size": 9,
+                "lastModified": datetime.datetime.now(),
+            },
+            {
+                "resourceId": 2,
+                "data": 1,
+                "name": "file_name_2",
+                "checksum": str(uuid.uuid1()),
+                "size": 3,
+                "uri": "file_uri_2",
+                "lastModified": datetime.datetime.now(),
+            }
+        ]
+        self.data["uriVariableValue"] = [
+            {"uriVariableId": 1, "resource": 1, "value": "uripath"}
+        ]
+
+    def test_view_home(self):
+        response = self.client.get(reverse('home'))
+        self.assertEqual(response.status_code, 200, msg="The home page is not returning a 200 status code")
+
+    def test_view_my_workflows_authenticated(self):
+        response = self.client.get(reverse('my_workflows'))
+        self.assertEqual(response.status_code, 200,
+                         msg="An authanticated user was unable to access my_workflows page")
+
+    def test_view_my_workflows_unauthenticated(self):
+        client = APIClient()
+        username = str(uuid.uuid1())
+        password = "Password!@#"
+        user = User.objects.create_user(username=username, password=password)
+
+        res = client.post('/rest-auth/login/', data={'username': username, 'password': password},
```

Comment:
we're testing an unauthenticated client, so  we shouldn't log in. that's tested in the above test